examples/CSharp/Tutorial 1.6 - Hello Gamepad/Tutorial 1.6 - Hello Gamepad.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can you:
- merge in the latest `main`
- move this project to `examples/CSharp/HLU Demos/Gamepad Input Demo`